src/views/Home/components/FAQ/index.scss

```diff
@@ -0,0 +1,75 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.faq-component {
+  margin-top: -1px;
+  background-size: cover;
+  background: linear-gradient(
+    var(--color-light-pink),
+    var(--color-lavender),
+    var(--color-sky-blue)
+  );
+  &__box {
+    margin: auto;
+    border: 7px solid rgba(255, 255, 255, 0.75);
+    border-radius: 59px;
+    padding: 10px;
+    display: table;
+    // height: 1400px;
+    width: 90%;
+    @include responsiveness.screen(large-tablet, phone) {
+      width: 50%;
+    }
+  }
+  &__faqtitle {
+    font-family: "Source Sans Pro", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    @include responsiveness.font($responsive: 4.1vw, $min: 35px, $max: 60px);
```

Comment:
This needs either a margin-bottom or a line-height attached to this

Reply:
fixed